frontend/packages/pipelines-plugin/src/components/import/pipeline/PipelineTemplate.tsx

```diff
@@ -118,6 +171,14 @@ const PipelineTemplate: React.FC<PipelineTemplateProps> = ({ builderImages, exis
         name="pipeline.enabled"
         isDisabled={isPipelineAttached}
       />
+      {pipeline.enabled && (
+        <DropdownField
+          name="pipeline.templateSelected"
+          items={pipelineTemplateItems}
```

Comment:
This should solve the empty dropdown issue when resource type is altered. Refer my previous comment

```suggestion
          title={pipelineTemplateItems[pipeline.templateSelected]}
          items={pipelineTemplateItems}
```

Reply:
thank you it worked :D